### ISSUE TITLE
storage: node liveness records shouldn't use MVCC

### ISSUE THREAD

**Issue by andreimatei** [cockroachdb/cockroach#37708]
These records currently use MVCC, but that seems ridiculous. We're wasting space and causing splits in a sensitive region of the key space.
I think we should move away from MVCC, but I'm not entirely sure about how the migration would work. @tbg have you ever thought about this?

I've also found this random note that wants the liveness record key prefix changed, which perhaps could be done at the same time.
https://github.com/cockroachdb/cockroach/issues/18276#issuecomment-327925691

cc @nvanbenschoten

**Comment by tbg** [comment]
I agree that MVCC isn't useful here. My expectation is that we'll move away from node liveness (for example to @andy-kimball's proposed failure detector) and that it won't be worth it to do something in the interim (was this issue motivated by something, for example a stability problem? I know we've had some in the past, but thought we had successfully worked around them for now).
Which splits do you mean? The liveness range itself can't be split.

**Comment by andreimatei** [comment]
> was this issue motivated by something, for example a stability problem?

No, it wasn't motivated by anything in particular.

> Which splits do you mean? The liveness range itself can't be split.

I was actually wondering about that. I thought it wouldn't, but then I couldn't find what prevents it. Now I did.

**Comment by knz** [comment]
definitely still relevant

**Comment by knz** [comment]
cc @andrewbaptist
